hazelcast/src/main/java/com/hazelcast/client/config/ClientConfig.java

```diff
@@ -168,6 +172,45 @@ public ClientConfig(ClientConfig config) {
         metricsConfig = new ClientMetricsConfig(config.metricsConfig);
```

Comment:
Do we log which file got loaded? So that at least from the logging it is obvious which file is being used. It can be extremely annoying if you are fighting an configuration issue and then you finally determine that your file didn't get loaded in the first place.

Reply:
I think that we do log it. At least I see something like this in the logs:
```
INFO: Loading configuration '/opt/hazelcast/hazelcast.yaml' from System property 'hazelcast.config'
```